pulp_smash/cli.py

```diff
@@ -722,6 +731,25 @@ def name(self):
             self._name = self._get_package_manager(self._cfg)
         return self._name
 
+    def raise_for_status(self, exc, message='Unsupported package manager'):
```

Comment:
I would like to suggest for us to update this function name. `raise_for_status` implies that a certain status was verified and that  is the reason for failure. Perhaps, I am so used to this function name in the context of a HTTP request. Just a suggestion.

Reply:
@kersommoura naming is hard :( 

```
This is inspired by `requests.raise_for_status` and in this
        case status can be `Known` or `Not Known` package manager.
```

I named `raise_for_status` as we already know what the function does because of requests, in this case the implicit status would be `supported` and `unsupported` package manager.

I had no other idea to name this function, do you have a suggestion?

- raise_for_support
- raise_if_unsupported
- raise_on_error